cf/ses.yaml

```diff
@@ -0,0 +1,51 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Set up SES for Cellenics [managed by github.com/hms-dbmi-cellenics/iac]
+
+Parameters:
+  DomainName:
+    Type: String
+    Description: Domain name of the deployment (e.g. scp.biomage.net)
+  PrimaryDomainName:
+    Type: String
+    Description: Primary domain name of the deployment (e.g. biomage.net)
+  Environment:
+    Type: String
+    Default: development
+    AllowedValues:
+      - development
+      - staging
+      - production
+    Description: The environment for which the SES records need to be created.
+
+Conditions:
+  isProd: !Equals [!Ref Environment, "production"]
+
+Resources:
+  SESVerifiedIdentity:
+    Type: AWS::SES::EmailIdentity
+    Properties:
+      EmailIdentity: !If [ isProd, !Ref DomainName, !Sub "scp-staging.${PrimaryDomainName}" ]
```

Comment:
`scp-staging` is Biomage specific

What is SES being used for?

Reply:
SES - Simple Email Service by Amazon, we use it to send emails. In the HMS IAC the SES template didn't exist, so we assumed you don't use it (also added in excluded_services for HMS in `deployment_config.json`).

Reply:
Is it just for the emails that are sent through the API? I see that it was added 2 months ago - was the sending email address changed? Just trying to understand why this was necessary and whether or not HMS needs it. I believe I excluded it as it wasn't deploying properly and it wasn't clear to me that it was necessary for HMS purposes. In either case, the service should not use hard-coded organization specific values (`scp-staging`). Maybe move to   `deployment-config.json`?

Reply:
Great - thank you for the explanation. Can the staging value (`scp-staging.${PrimaryDomainName}`) be changed to something that isn't Biomage specific and doesn't assume that the domain is a subdomain (true for Biomage, not for HMS). I suspect that this may be why the deployment originally failed.